worldwind/src/commonMain/kotlin/earth/worldwind/draw/DrawContext.kt

```diff
@@ -35,6 +36,9 @@ open class DrawContext(val gl: Kgl) {
     private var unitSquareBufferCache: FloatBufferObject? = null
     private var scratchBuffer = ByteArray(4)
     private val pixelArray = ByteArray(4)
+
+    //Buffer pool for line vertex data
+    private var lineBufferPool : StorageBufferPool = StorageBufferPool(GL_ARRAY_BUFFER, GL_DYNAMIC_DRAW)
```

Comment:
In OpenGL Storage Buffer has specific meaning - SSBO. I would consider another name. Maybe just BufferPool

Reply:
done